src/gradient/sensitivity_calculator.py

```diff
@@ -1,27 +1,47 @@
+from src.static.cm_data_aggregate_kenya import KenyaDataAggregator
```

Comment:
Sort all imports alphabetically (since imports are only from this project here).